app/interactors/update_project.rb

```diff
@@ -0,0 +1,7 @@
+class UpdateProject
+	include Interactor::Organizer
+
+	organize UpdateProject::PrepareParams,
+           UpdateProject::SaveRecord,
+           # UpdateProject::SendNotifications
```

Comment:
why commented?
And "," in the line before is a syntax error in this case

Reply:
i dont save change in this file, my bad